gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Apr 22 15:05:06 MDT 2015
+#Mon Apr 04 14:20:24 MDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.12-bin.zip
```

Comment:
3.0,3.1 is out now. The only changes I had that weren't compatible was needing to change the directory Jenkins looks for junit report XMLs.